<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search: %1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gb_search">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>32767</height>
        </size>
       </property>
       <property name="title">
        <string>Search Fields</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lb_instructions">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>Instructions go here.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>400</width>
               <height>400</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="IconButton" name="pb_search" native="true">
              <property name="text" stdset="0">
               <string>&amp;Search</string>
              </property>
              <property name="psiIconName" stdset="0">
               <string>psi/search</string>
              </property>
              <property name="default" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Spacer1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>30</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="IconButton" name="pb_stop" native="true">
              <property name="text" stdset="0">
               <string>&amp;Stop</string>
              </property>
              <property name="psiIconName" stdset="0">
               <string>psi/stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="lv_results">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="indentation">
          <number>0</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Nickname</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>First Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>E-Mail Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>XMPP Address</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="Spacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>222</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="IconButton" name="pb_add" native="true">
           <property name="text" stdset="0">
            <string>&amp;Add Contact</string>
           </property>
           <property name="psiIconName" stdset="0">
            <string>psi/addContact</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="IconButton" name="pb_info" native="true">
           <property name="text" stdset="0">
            <string>User &amp;Info</string>
           </property>
           <property name="psiIconName" stdset="0">
            <string>psi/vCard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="BusyWidget" name="busy" native="true"/>
     </item>
     <item>
      <spacer name="Spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>485</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_close" native="true">
       <property name="text" stdset="0">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BusyWidget</class>
   <extends>QWidget</extends>
   <header>busywidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
